cmd/contour/contour.go

```diff
@@ -97,13 +99,16 @@ func main() {
 	case serve.FullCommand():
 		// parse args a second time so cli flags are applied
 		// on top of any values sourced from -c's config file.
-		_, err := app.Parse(args)
-		check(err)
+		if _, err := app.Parse(args); err != nil {
```

Comment:
This one should probably use `kingpin.MustParse(app.Parse(args))` instead of a full `if err != nil` construct.

That will have Kingpin exit if the parsing fails.

Reply:
oops, thanks. You're right. Leave this one as is.